src/test/java/com/ifactory/press/db/solr/spelling/suggest/MultiSuggesterTest.java

```diff
@@ -255,26 +283,31 @@ public void testExtendedResultFormat() throws Exception {
         q.setRequestHandler("/suggest/all");
         QueryResponse resp = solr.query(q);
         SpellCheckResponse scr = resp.getSpellCheckResponse();
-        //Suggestion suggestion = scr.getSuggestion("t");
+        
 
         // no extended results
         assertNull(scr);
-
+        q = new SolrQuery("th");

```

Comment:
sorry when I said use `th` I was saying at the insert level instead of `t1` but it is up to you. Just needs to be consistent.

Reply:
the consistency before was artificial.  It was just the the t(num) and their and time started with t's. It's really no different except I had to force it in a different direction since the t1 doesn't continue on to 'their'.  I thought bought respelling their to t1heir but I thought that would be a little 'on the nose' 😄

Reply:
Gotcha